Meta/Lagom/Tools/CodeGenerators/LibWeb/GenerateCSSPropertyID.cpp

```diff
@@ -661,17 +661,20 @@ bool property_affects_stacking_context(PropertyID property_id)
     }
 }
 
-NonnullRefPtr<CSSStyleValue> property_initial_value(JS::Realm& context_realm, PropertyID property_id)
+NonnullRefPtr<CSSStyleValue> property_initial_value(Optional<JS::Realm&> context_realm, PropertyID property_id)
 {
     static Array<RefPtr<CSSStyleValue>, to_underlying(last_property_id) + 1> initial_values;
     if (auto initial_value = initial_values[to_underlying(property_id)])
         return initial_value.release_nonnull();
 
+    // We need a Realm to parse any new values.
+    VERIFY(context_realm.has_value());
```

Comment:
@AtkinsSJ we hit this assertion in the Text/input/wpt-import/css/cssom/serialize-values.html test (I was going to re-enable it now that tests are faster, but it crashes here now).
```
1   liblagom-ak.0.0.0.dylib       	       0x100c5337c ak_verification_failed + 176 (Assertions.cpp:110)
2   liblagom-web.0.0.0.dylib      	       0x10283a2f4 Web::CSS::property_initial_value(AK::Optional<JS::Realm&>, Web::CSS::PropertyID) + 28820 (PropertyID.cpp:5837)
3   liblagom-web.0.0.0.dylib      	       0x1023ba7a4 Web::CSS::ShorthandStyleValue::to_string() const::$_3::operator()(Web::CSS::PropertyID) const + 84 (ShorthandStyleValue.cpp:242)
4   liblagom-web.0.0.0.dylib      	       0x1023b8aa8 Web::CSS::ShorthandStyleValue::to_string() const + 5536 (ShorthandStyleValue.cpp:249)
5   liblagom-web.0.0.0.dylib      	       0x1022c4908 Web::CSS::CSSStyleDeclaration::get_property_value(AK::StringView) const + 676 (CSSStyleDeclaration.cpp:305)
```

Reply:
I opened #2720 with a smaller repro.